sonaranalyzer-dotnet/src/SonarAnalyzer.Common/Helpers/DiagnosticDescriptorBuilder.cs

```diff
@@ -44,11 +44,17 @@ public static class DiagnosticDescriptorBuilder
                 resourceManager.GetString($"{diagnosticId}_Category"),
                 ParseSeverity(resourceManager.GetString($"{diagnosticId}_Severity")).ToDiagnosticSeverity(ideVisibility),
                 bool.Parse(resourceManager.GetString($"{diagnosticId}_IsActivatedByDefault")),
-                helpLinkUri: diagnosticId.GetHelpLink(),
+                helpLinkUri: GetHelpLink(resourceManager, diagnosticId),
                 description: resourceManager.GetString($"{diagnosticId}_Description"),
                 customTags: ideVisibility.ToCustomTags());
         }
 
+        public static string GetHelpLink(ResourceManager resourceManager, string diagnosticId)
+        {
+            var helpLinkFormat = resourceManager.GetString("HelpLinkFormat");
+            return string.Format(helpLinkFormat, diagnosticId.Trim('S'));

```

Comment:
To be pedantic, it should only remove the first 'S', so perhaps use TrimStart (which is still not quite correct)?

This doesn't matter much, so feel free to ignore.